spec/serializers/solidus_klarna_payments/line_item_serializer_spec.rb

```diff
@@ -116,26 +116,25 @@
       end
 
       context "with a string as image_host" do
-        let(:image_host) { "example.com" }
+        let(:image_host) { "www.example.com" }
```

Comment:
Not sure to understand why we need to add `www.` Shouldn't this work without it as well?

Reply:
Yes and no, it would work if I set the Active Storage url to example.com but there were other examples using www.example.com. I just unified it for that purpose to work with AS

Reply:
Got it, makes sense, thanks!